src/templates/misc-templates.html

```diff
@@ -29,4 +29,43 @@
       {{description}}
     </div>
   </script>
+
+  <!-- ----------------------------------------------------------- -->
+  <!-- Resize Tool -->
+  <!-- Template for Resize Settings Title -->
+  <script type="text/template" id="resize-settings-template">
+    {{keyTitle}}
+  </script>
+
+  <!-- Template for Width and Height -->
+  <script type="text/template" id="resize-tool-template">
+    <div class="resize-section">
+      <span  class="resize-section-title">{{keyTitle}}</span>
```

Comment:
The "keyTitle" is sometimes just text and not a title. Consider using just `title` or `text` if possible.
Also, `cssClass` might be more clear than `keyCss`